.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,65 @@
+name: 🐞 Bug
+description: Report a bug or an issue you've found with dbt-oss-template
```

Comment:
I wonder if there's a way for us to do some substitution (i.e. swap out dbt-oss-template on the fly)

Reply:
_sigh..._ I guess we could always add a script file that does this for us